pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1426,6 +1490,496 @@ func TestReconcileOnCancelledPipelineRun(t *testing.T) {
 	}
 }
 
+func TestReconcileOnCancelledRunFinallyPipelineRun(t *testing.T) {
```

Comment:
thanks @rafalbigaj for adding comprehensive set of tests 🙏 

Can we change this test to have two pipelineTasks instead of one and making sure no new taskruns are created?

Reply:
Thanks for suggestion. The second task is in place.